### ISSUE TITLE
Database

### ISSUE THREAD

**Issue by rynobax** [rynobax/hathi#3]
What should we use for the database? Probably something free 😀

Free hosted options I know of:

Firestore
Prisma cloud
Dynamodb
We could also run a free server on ec2 or something, but that sounds like a pain

**Comment by rynobax** [comment]
I've always wanted to try Prisma so I would vote that

**Comment by bharget** [comment]
Prisma works for me! I was actually planning on using it for something else with MySQL and GraphQL Yoga

**Comment by bharget** [comment]
I think we would also need to pick a database type right? Like MySQL or PostgreSQL. I think I would opt for MySQL but I'm open to trying something different

**Comment by rynobax** [comment]
Yeah.  We should probably decide on the general form of the data before we pick one
